pages/api/check-status.ts

```diff
@@ -94,7 +70,7 @@ export const fetchPassportStatusAPI = async (
   )
 
   if (response.ok) {
-    const passportStatusesResponse = (await response.json()) as PassportStatuses
+    const passportStatusesResponse = await response.json() //) as CheckStatusReponse[]
```

Comment:
Returned JSON from API should be mapped with TS Type otherwise it will be type of `any` which is discouraged. If the API change the JSON structure then you can't easily validate this change in your code. If you have a type that match the API then updated it will ensure your code reflect the changes.

Reply:
In complex calls where there is parsing of the data and not everything gets passed on I can completely understand. But the objects are nearly identical, short of being nested in the return object. This API is already doing way more than it should; all it should be is a (redundant) pass though.

Reply:
1. You've convinced me... 
2. Over zealous ESDC IT Security... (a client browser should not have rights to call an "internal" api - even if in this case that API is open - it's firmly against our architecture standards)

Reply:
Added the Types back.